src/services/ws.ts

```diff
@@ -123,22 +142,24 @@ function fillInAdditionalProperties(entityChange) {
         if (!entityChange.entity) {
             entityChange.entity = sql.getRow(`SELECT * FROM notes WHERE noteId = ?`, [entityChange.entityId]);
 
-            if (entityChange.entity.isProtected) {
-                entityChange.entity.title = protectedSessionService.decryptString(entityChange.entity.title);
+            if (entityChange.entity && entityChange.entity.isProtected) {
```

Comment:
What version of node is this? Do we have optional chaining?

```suggestion
            if (entityChange.entity?.isProtected) {
```

Reply:
Good find.